### PR TITLE
LRQA-73645 | master

### PR DESCRIPTION
**Pull request by melodywu**
https://issues.liferay.com/browse/LRQA-73645

### CODE REVIEW

**Comment by melodywu** [comment]
ci:test:sf

**Comment by Sharryxue** [comment]
Just starting review.

File: modules/apps/account/account-test/src/testFunctional/macros/AccountGroups.macro [hunk 1]
```diff
@@ -115,6 +115,29 @@ definition {
 		LexiconEntry.viewNoEntry(rowEntry = "${accountGroupName}");
 	}
 
+	macro unassign {
```
Comment (Sharryxue) [inline modules/apps/account/account-test/src/testFunctional/macros/AccountGroups.macro]: This should be duplicate with the exist macro, please double check it, thanks.
Always remember to check the exist macro first before create a new one.

**Review by Sharryxue — CHANGES_REQUESTED** [review]
Hello @melodywu ,
Please use the exist macro instead. Thanks.

**Comment by melodywu** [comment]
> LRQA-73645 Add testcase to unassign account from group

Done thanks a lot

**Comment by melodywu** [comment]
ci:test:sf

**Comment by Sharryxue** [comment]
Just starting review.

**Comment by Sharryxue** [comment]
No additional changes needed.

**Comment by Sharryxue** [comment]
Rebased and resend [Sharryxue/liferay-portal#396](https://github.com/Sharryxue/liferay-portal/pull/396).

**Pull request closed without merging**